Packs/CVE_2022_41040_and_CVE_2022_41082_-_ProxyNotShell/Playbooks/playbook-CVE-2022-41040_&_CVE-2022-41082_-_ProxyNotShell.yml

```diff
@@ -0,0 +1,3089 @@
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 3eacb1f5-526c-4195-8917-70a9b8c55461
+    type: title
+    task:
+      id: 3eacb1f5-526c-4195-8917-70a9b8c55461
+      version: -1
+      name: SIEM Advanced Hunting
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "60"
+      - "61"
+      - "62"
+      - "70"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -310,
+          "y": 1000
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: d4e09e34-a17b-4dc0-8e7f-75c05d7416c7
+    type: regular
+    task:
+      id: d4e09e34-a17b-4dc0-8e7f-75c05d7416c7
+      version: -1
+      name: Detect suspicious ASPX file dropped by Exchange
+      description: |-
+        Detects suspicious file types dropped by an Exchange component in IIS into a suspicious folder.
+
+        **Author:** Florian Roth (rule), MSTI (query, idea)
+
+        **Status:** experimental
+
+        **References:**
+           - https://www.microsoft.com/security/blog/2022/09/30/analyzing-attacks-using-the-exchange-vulnerabilities-cve-2022-41040-and-cve-2022-41082/
+           - https://www.gteltsc.vn/blog/canh-bao-chien-dich-tan-cong-su-dung-lo-hong-zero-day-tren-microsoft-exchange-server-12714.html
+           - https://en.gteltsc.vn/blog/cap-nhat-nhe-ve-lo-hong-bao-mat-0day-microsoft-exchange-dang-duoc-su-dung-de-tan-cong-cac-to-chuc-tai-viet-nam-9685.html
+      tags:
+      - SIEMResults
+      script: '|||splunk-search'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "17"
+    scriptarguments:
+      earliest_time:
+        complex:
+          root: inputs.SplunkEarliestTime
+      latest_time:
+        complex:
+          root: inputs.SplunkLatestTime
+      query:
+        simple: source="WinEventLog:*" AND (Image="*\\w3wp.exe" AND CommandLine="*MSExchange*" AND (TargetFilename="*FrontEnd\\HttpProxy\\*" OR TargetFilename="*\\inetpub\\wwwroot\\aspnet_client\\*") AND (TargetFilename="*.aspx" OR TargetFilename="*.asp" OR TargetFilename="*.ashx"))
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -90,
+          "y": 1400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 8ac20155-372a-4b63-8053-5900058c4d69
+    type: playbook
+    task:
+      id: 8ac20155-372a-4b63-8053-5900058c4d69
+      version: -1
+      name: QRadarFullSearch
```

Comment:
lets change to the name of the task (and not the subplaybook name) for all tasks